Signal/src/AppDelegate.m

```diff
@@ -230,35 +230,64 @@ - (void)verifyDBKeysAvailableBeforeBackgroundLaunch
 
 - (BOOL)ensureIsReadyForAppExtensions
 {
+    // Given how sensitive this migration is, we verbosely
+    // log the contents of all involved paths before and after.
+    //
+    // TODO: Remove this logging once we have high confidence
+    // in our migration logic.
+    NSArray<NSString *> *paths = @[
+        TSStorageManager.legacyDatabaseFilePath,
```

Comment:
Fine for now, but we have to be sure that none of these class methods ever instantiate an instance of TSStorageManager, like some of our class methods do.

Reply:
+1